agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/LoggingConfigurator.java

```diff
@@ -94,12 +94,10 @@ private void configureAppSvc() {
     rootLogger.addAppender(configureFileAppender());
     rootLogger.addAppender(configureConsoleAppender());
 
-    // App Services linux environments set this environment variable to control where the internal
-    // diagnostic log is written, so that App Services can consume that file and send those logging
-    // events to an internal kusto store for internal alerting and diagnostics
-    String diagnosticsOutputDirectory =
-        System.getenv(DiagnosticsHelper.APPLICATIONINSIGHTS_DIAGNOSTICS_OUTPUT_DIRECTORY);
-    if (diagnosticsOutputDirectory != null && !diagnosticsOutputDirectory.isEmpty()) {
+    // App Services linux is default to "/var/log/applicationinsights".
+    // they ignore the value set in env var APPLICATIONINSIGHTS_DIAGNOSTICS_OUTPUT_DIRECTORY)
```

Comment:
do we use `APPLICATIONINSIGHTS_DIAGNOSTICS_OUTPUT_DIRECTORY` for anything else? if not, then I think we can delete the constant and remove this comment